episodes/ingredients.md

```diff
@@ -5,3 +5,29 @@ exercises: 0
 ---
 
 Placeholder text
+
+::: discussion
+
+### Let's talk about muffins
+
+Prompt for the learners to discuss.
+
```

Comment:
@sstevens2 - could you please add a solution box for the discussion to provide guidance for instructors?

Reply:
Added. What do you think?